Wikipedia/Code/WMFArticleFooterMenuViewController.m

```diff
@@ -39,6 +39,16 @@ - (void)viewWillAppear:(BOOL)animated {
     [self.tableView deselectRowAtIndexPath:[self.tableView indexPathForSelectedRow] animated:YES];
 }
 
+- (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSInteger)section {
```

Comment:
ah, gotchya. i knew we had discussed this before, but forgot. would this also work if we set `sectionHeaderHeight` and `sectionFooterHeight` properties in `viewDidLoad`?  might also be worth DRYing this up into a category and explaining why we do this in doc comments.